<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Raw_image_dialog</class>
 <widget class="QDialog" name="Raw_image_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open raw image</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Image &amp;value type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QRadioButton" name="short_bt">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Short int (16 bits)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="float_bt">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Float</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="int_bt">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Int (32 bits)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="double_bt">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Double</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="signed_bt">
        <property name="toolTip">
         <string>Specifies if the image's word kind is signed.</string>
        </property>
        <property name="text">
         <string>Signed</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="char_bt">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>char (8Â bits)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Image dimensions</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_dim">
          <property name="toolTip">
           <string>Number of voxel along the axis.</string>
          </property>
          <property name="text">
           <string>&amp;Dimensions:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>dim_x</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_spacing">
          <property name="toolTip">
           <string>Changing those values will scale the image according to the proportions between vx, vy and vz.</string>
          </property>
          <property name="text">
           <string>&amp;Spacing:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>spacing_x</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="dim_x">
          <property name="toolTip">
           <string>Number of voxels along the X axis.</string>
          </property>
          <property name="prefix">
           <string>x: </string>
          </property>
          <property name="maximum">
           <number>100000000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="dim_y">
          <property name="toolTip">
           <string>Number of voxels along the Y axis.</string>
          </property>
          <property name="prefix">
           <string>y: </string>
          </property>
          <property name="maximum">
           <number>100000000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QSpinBox" name="dim_z">
          <property name="toolTip">
           <string>Number of voxels along the Z axis.</string>
          </property>
          <property name="prefix">
           <string>z: </string>
          </property>
          <property name="maximum">
           <number>100000000</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="spacing_x">
          <property name="toolTip">
           <string>Voxel size in x.</string>
          </property>
          <property name="prefix">
           <string>vx: </string>
          </property>
          <property name="decimals">
           <number>5</number>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QDoubleSpinBox" name="spacing_y">
          <property name="toolTip">
           <string>Voxel size in y.</string>
          </property>
          <property name="prefix">
           <string>vy: </string>
          </property>
          <property name="decimals">
           <number>5</number>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QDoubleSpinBox" name="spacing_z">
          <property name="toolTip">
           <string>Voxel size in z.</string>
          </property>
          <property name="prefix">
           <string>vz: </string>
          </property>
          <property name="decimals">
           <number>5</number>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="offset_label">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>&amp;Offset:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>offset</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="offset">
          <property name="toolTip">
           <string>Specifies how many bytes must be skipped before reading the data. (Use it to skip unhandled headers for example.)</string>
          </property>
          <property name="suffix">
           <string> bytes</string>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="imagesize_budy">
          <property name="text">
           <string>Image size:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_image_size">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="filesize_budy">
          <property name="text">
           <string>File size:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_file_size">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Open</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>int_bt</tabstop>
  <tabstop>short_bt</tabstop>
  <tabstop>signed_bt</tabstop>
  <tabstop>float_bt</tabstop>
  <tabstop>double_bt</tabstop>
  <tabstop>dim_x</tabstop>
  <tabstop>dim_y</tabstop>
  <tabstop>dim_z</tabstop>
  <tabstop>spacing_x</tabstop>
  <tabstop>spacing_y</tabstop>
  <tabstop>spacing_z</tabstop>
  <tabstop>offset</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>float_bt</sender>
   <signal>toggled(bool)</signal>
   <receiver>signed_bt</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>553</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>double_bt</sender>
   <signal>toggled(bool)</signal>
   <receiver>signed_bt</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>564</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Raw_image_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Raw_image_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dim_x</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Raw_image_dialog</receiver>
   <slot>update_image_size()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dim_z</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Raw_image_dialog</receiver>
   <slot>update_image_size()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>char_bt</sender>
   <signal>clicked()</signal>
   <receiver>Raw_image_dialog</receiver>
   <slot>update_image_size()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>short_bt</sender>
   <signal>clicked()</signal>
   <receiver>Raw_image_dialog</receiver>
   <slot>update_image_size()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>float_bt</sender>
   <signal>clicked()</signal>
   <receiver>Raw_image_dialog</receiver>
   <slot>update_image_size()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>-5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>int_bt</sender>
   <signal>clicked()</signal>
   <receiver>Raw_image_dialog</receiver>
   <slot>update_image_size()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>584</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dim_y</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Raw_image_dialog</receiver>
   <slot>update_image_size()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>double_bt</sender>
   <signal>clicked()</signal>
   <receiver>Raw_image_dialog</receiver>
   <slot>update_image_size()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offset</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Raw_image_dialog</receiver>
   <slot>update_image_size()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>update_image_size()</slot>
 </slots>
</ui>
